knowledge/miscellaneous_unknown/tarcento/feasts/attribution.txt

```diff
@@ -0,0 +1,3 @@
+Title of work: Pignarûl Grant
+License of the work: CC-BY-SA-4.0
+Creator names: Maja Massarini
```

Comment:
I'm sorry but we are taking not taking submissions like this. Until we have clear direction from legal please put a PR with this domain in this: https://github.com/instructlab/dev-docs/blob/main/docs/knowledge-submissions-past-wikipedia.md